app/controllers/items/new.js

```diff
@@ -34,6 +48,64 @@ export default GoodcityController.extend({
   imageKeys: Ember.computed.localStorage(),
 
   i18n: Ember.inject.service(),
+
+  insertFixedOptions: function(column) {
+    let package_details = this.get("packageDetails");
+    switch (column) {
+      case "frequency":
+        return ["50", "50/60 (Multi)", "60", "N/A", "Other"];
+      case "voltage":
+        return [
+          "120V ~ (100-200V)",
+          "240V ~ (200-300V)",
+          "> 300V",
+          "Multi",
+          "NA",
+          "Other"
+        ];
+      case "testStatus":
+        return [
+          "Maintenance (DO NOT USE)",
+          "Tested (DO NOT USE)",
+          "Untested (DO NOT USE)"
+        ];
+      case "compTestStatus":
+        return ["Active", "Failure", "Obsolete", "Reserved", "Spares"];
+      default:
+        return [...new Set(package_details.getEach(column).filter(Boolean))];
+    }
+  },
```

Comment:
Already defined same in another controller `app/controllers/items/detail.js`. Move to some other common location like `GoodcityController` or use mixin/service